### ISSUE TITLE
TPKeyboardAvoidingTableView not working when using Sougou Chinese input on iOS8

### ISSUE THREAD

**Issue by mykoma** [michaeltyson/TPKeyboardAvoiding#136]
I met this bug when I used the TPKeyboardAvoidingTableView in storyboard, and I found it's caused by the third party input -- Sougou Chinese input (搜狗拼音输入). 

The code in UIScrollView+TPKeyboardAvoidingAdditions.m Line 51.

```
    state.keyboardRect = [self convertRect:[[[notification userInfo] objectForKey:_UIKeyboardFrameEndUserInfoKey] CGRectValue] fromView:nil];
```

this rect is not the all keyboard bounds, the height is 44. 

I am not sure this bug happened or not on TPKeyboardAvoidingCollectionView, 

Hope this information could help others.

**Comment by Tinghui** [comment]
This has been fixed in version 1.2.9

**Comment by michaeltyson** [comment]
Yay!
